drivers/flash/Kconfig.mcux

```diff
@@ -11,3 +11,16 @@ config SOC_FLASH_MCUX
 	  the duration of the flash erase/write operations. This will
 	  have an impact on the overall system performance - whether
 	  this is acceptable or not will depend on the use case.
+
+if SOC_FLASH_MCUX
+
+config SW_MANAGED_FLASH_WRITE
```

Comment:
It is still unclear to me what the use case is here? This should never be defined.